core/packages/contracts/scripts/configure-beefy.ts

```diff
@@ -9,17 +7,13 @@ import type {
     BeefyNextAuthoritySet,
     BeefyId,
 } from "@polkadot/types/interfaces/beefy/types"
+import fs from "fs"
 
 let endpoint = process.env.RELAYCHAIN_ENDPOINT || "ws://localhost:9944"
 const beefyStartBlock = process.env.BEEFY_START_BLOCK ? parseInt(process.env.BEEFY_START_BLOCK) : 15
+const BeefyStateFile = process.env.BEEFY_STATE_FILE || "/tmp/snowbridge/beefy-state.json"
```

Comment:
Let's use `OUTPUT_DIR` here too, in case we change it:
```suggestion
const BeefyStateFile = process.env.BEEFY_STATE_FILE || path.join(process.env.OUTPUT_DIR, "beefy-state.json")
```